systemtest/src/test/java/org/bf2/systemtest/unit/SuiteUnitTest.java

```diff
@@ -42,9 +49,16 @@
  */
 @QuarkusTest
 @QuarkusTestResource(KubernetesServerTestResource.class)
-public class SuiteUnitTest extends AbstractST {
```

Comment:
What was the motivation to remove the base class?

Reply:
Aside from the annotations, the only thing gained was the setup of the `KubeClient`. I figured the three other *ST classes had more to gain by consolidating setup in the base class without adding an intermediate parent.